### DISCUSSION TITLE
CCDA references

### DISCUSSION THREAD

**Discussion by smathur-ith** [nextgenhealthcare/connect#4625] [Q&A] [answered]
Hi All,

Can anyone provide reference/samples for working with CCDA documents in MIRTH? We are using Mirth 3.9 but community version.

**Comment by pacmano1** [comment, marked as answer]
If you don't use the platinum licensed version of Mirth which has the CDAPI library for CDAs, folks use MDHT I believe or parse the raw XML with normal e4x functions.  There are a couple of other libraries out there as well.

An example of Mirth using MDHT is at https://www.spheregen.com/parsing-continuity-of-care-ccd-using-mirth-and-mdht/.

I am not sure you will find much help on finding references for CDA creation or consumption given both usually involve a large body of work that someone has paid for.   Best wishes!

**Reply by smathur-ith** [reply]
Thanks @pacmano1 for quick response! yeah MDHT is the only thing that came up in my searches too. I think I am going to explore more on that.
